modules/utxo-lib/test/bitgo/litecoin/mwebBehavior.ts

```diff
@@ -0,0 +1,102 @@
+import * as assert from 'assert';
+
+import * as path from 'path';
+import * as fs from 'fs-extra';
+
+import { networks } from '../../../src';
+import {
+  addWalletUnspentToPsbt,
+  createPsbtForNetwork,
+  createPsbtFromTransaction,
+  createTransactionFromBuffer,
+  createTransactionFromHex,
+  LitecoinPsbt,
+  LitecoinTransaction,
+  NonWitnessWalletUnspent,
+} from '../../../src/bitgo';
+import { getDefaultWalletKeys, mockWalletUnspent } from '../../../src/testutil';
+
+const network = networks.litecoin;
+
+async function getFixture<T>(name: string): Promise<T> {
```

Comment:
Has this been tested to make sure it works properly in a web environment like the `web-demo` module?